go/interpreter/lfvm/keccak.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package lfvm
+
+/*
+#include "keccak.h"
+*/
+import "C"
+
+import (
+	"sync"
+	"unsafe"
+
+	"github.com/Fantom-foundation/Tosca/go/tosca"
+	"golang.org/x/crypto/sha3"
+)
+
+func Keccak256(data []byte) tosca.Hash {
+	return keccak256_C(data)
+}
+
+func Keccak256ForKey(key tosca.Key) tosca.Hash {
```

Comment:
We do not need it for the `tosca.Key` but for a generic `[32]byte` array.

Reply:
done